sklearn/tree/_classes.py

```diff
@@ -1203,12 +1208,10 @@ def fit(self, X, y, sample_weight=None, check_input=True,
             Allow to bypass several input checking.
             Don't use this parameter unless you know what you do.
 
-        X_idx_sorted : array-like of shape (n_samples, n_features), \
-            default=None
-            The indexes of the sorted training input samples. If many tree
-            are grown on the same dataset, this allows the ordering to be
-            cached between trees. If None, the data will be sorted here.
-            Don't use this parameter unless you know what to do.
+        X_idx_sorted : deprecated, default="deprecated"
+            This parameter is deprecated and will be removed in v0.26.
```

Comment:
You can mention that it already has no effect.

Reply:
Here and above

Reply:
Done!